src/accessories/alarm.js

```diff
@@ -201,10 +200,17 @@ class SS3Alarm {
                             this.service.updateCharacteristic(this.Characteristic.SecuritySystemTargetState, this.Characteristic.SecuritySystemTargetState.AWAY_ARM);
                             break;
                         case EVENT_TYPES.DISCONNECT:
-                            this.log('Real time events disconnected.');
+                            this.log('Alarm real time events disconnected.');
+                            break;
+                        case EVENT_TYPES.RECONNECT:
+                            this.log('Alarm real time events re-connected.');
+                            break;
+                        case EVENT_TYPES.RECONNECT_FAILED:
+                            this.log(this.name + ' camera real time events re-connect failed. Restarting');
```

Comment:
Change this log to `this.log(this.name + ' alarm real time events re-connect failed. Restarting');`